docs/UserGuide.md

```diff
@@ -175,13 +176,28 @@ Examples:
 * `addtask d/Remind Alice to change bedsheets. r/5` Adds a task with description "Remind Alice to change bedsheets." to Room #5.
 * `addtask d/Running low on masks and needs to be restocked. r/1 dd/12-1-2021` Adds a task with description "Running low on masks and needs to be restocked." and due date "12 Jan 2021 at 12am" to Room #1.
 
-### 3.9 View help: `help`
+### 3.9 Search tasks with a deadline: `searchtask`
+
+Search all tasks before a date.
```

Comment:
Can consider "Searches all tasks before the given date."

Reply:
Sure!